CMakeLists.txt

```diff
@@ -46,5 +56,4 @@ endif(WIN32)
 
 # install
 install(TARGETS run-swmm DESTINATION bin)
-install(TARGETS swmm5 DESTINATION lib)
-install(FILES src/swmm5.h DESTINATION include)
```

Comment:
@bemcdonnell this is pretty standard and might be needed for any other program linking to swmm, so why erasing it? (For conda is quite useful)